setup.cfg

```diff
@@ -178,7 +178,7 @@ exclude =
     doc/source/getting_started/comparison/includes/*.rst
 
 [codespell]
-ignore-words-list = ba,blocs,coo,hist,nd,sav,ser
+ignore-words-list = ba,blocs,coo,hist,nd,sav,ser,bu,ro,recuse,fromm
```

Comment:
bu was a concatenated string on pandas/tests/arrays/string_/test_string.py:185
ro was an abbreviation for registered options on pandas/tests/config/test_config.py:19 and pandas/tests/config/test_config.py:36
fromm was a way of avoiding reserved words on pandas/io/parsers/python_parser.py:1275

Reply:
can we turn `fromm` to `from_`, to avoid having to ignore a word?

would it work to rename `ro` to `_registered_options`?

for `bu` - could the strings to be concatenated be changed slightly, so we don't hit any potential typos?

Reply:
Sure thing, i will try to solve those removing the words from the ignore list. Thanks for the feedback.

Reply:
No worries

And I've opened a PR to codespell about recuse because it's a real word